bin/wcs.js

```diff
@@ -108,16 +117,25 @@ if (options.help) {
 if (options.root !== '' && !/[\/\\]$/.test(options.root)) {
   options.root += '/';
 }
-var workPath = path.resolve(options.workdir);
+if (options.dest_dir !== '' && !/[\/\\]$/.test(options.dest_dir)) {
+    options.dest_dir += '/';
+}
+if (options.workdir !== '' && !/[\/\\]$/.test(options.workdir)) {
+    options.workdir += '/';
+}
+
+var workdirPath = url.resolve(options.root, options.workdir);
+var workPath = path.resolve(workdirPath);
 try {
   var workdir = fs.statSync(workPath);
   if (workdir) {
-    console.log("Working directory " + workPath + " already exists! Please clean up.");
-    process.exit(1);
+    rimraf.sync(workPath, {});
```

Comment:
Add a `--force` option that blows away the workdir. The default behavior shouldn't change so suddenly.